docusaurus/sidebars.js

```diff
@@ -453,6 +448,30 @@ const deployAirbyte = {
   ],
 };
 
+const connectionConfigurations = {
+  type: "category",
+  label: "Configuration Options",
```

Comment:
Do you have an alternative suggestion for "Configuration Options" so that it doesn't look similar to "Configuring Airbyte" below?

Reply:
Good reminder. Thinking about this now.